### DISCUSSION TITLE
Snippet: How to filter the output of a Block Binding

### DISCUSSION THREAD

**Discussion by ndiego** [WordPress/developer-blog-content#341] [Topic Ideas]
This snippet would show developers how to use the `get_value_callback` property of the [`register_block_bindings_source()`](https://developer.wordpress.org/reference/functions/register_block_bindings_source/) function and the [`block_bindings_source_value`](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-bindings/#server-registration) filter to modify the output of a Block Binding on the front end. This snippet will provide a solution for implementations like this: 

![image](https://github.com/user-attachments/assets/429cac7d-16e0-4cba-b489-9bbf02dd7c1b)

**Comment by bph** [comment]
This snippet was approved and conversation continues on the issue
- #345

**Comment by ndiego** [comment]
The snippet has been published here: https://developer.wordpress.org/news/snippets/how-to-filter-the-output-of-a-block-binding/
